db/python/layers/project_insights.py

```diff
@@ -301,367 +332,601 @@ async def _stats_es_indexes_query(
+        self, project_ids: list[ProjectId], sequencing_types: list[str]
+    ) -> defaultdict[tuple[ProjectId, str, str, str], list[dict]]:
         _query = """
-WITH mito AS (
-    SELECT DISTINCT asg.sequencing_group_id
-    FROM analysis a
-    LEFT JOIN analysis_sequencing_group asg on asg.analysis_id=a.id
-    WHERE status = 'COMPLETED'
-    AND type = 'WEB'
-    AND meta LIKE '%mitoreport%'
-),
-stripy AS (
-    SELECT DISTINCT asg.sequencing_group_id
-    FROM analysis a
-    LEFT JOIN analysis_sequencing_group asg on asg.analysis_id=a.id
-    WHERE status = 'COMPLETED'
-    AND type = 'WEB'
-    AND meta LIKE '%stripy%'
-)
-
-SELECT DISTINCT
-    a.project,
+SELECT
+    f.project,
     sg.type as sequencing_type,
-    sg.id as sequencing_group_id,
-    CASE WHEN stripy.sequencing_group_id IS NOT NULL THEN TRUE ELSE FALSE END as stripy,
-    CASE WHEN mito.sequencing_group_id IS NOT NULL THEN TRUE ELSE FALSE END as mito
+    sg.platform as sequencing_platform,
+    sg.technology as sequencing_technology,
+    s.type as sample_type,
+    f.id as family_id,
+    f.external_id as family_external_id,
+    fp.participant_id as participant_id,
+    p.external_id as participant_external_id,
+    s.id as sample_id,
+    s.external_id as sample_external_ids,
+    sg.id as sequencing_group_id
 FROM
-    sequencing_group sg
-    INNER JOIN analysis_sequencing_group asg ON sg.id = asg.sequencing_group_id
-    INNER JOIN analysis a ON asg.analysis_id = a.id
-    LEFT JOIN mito on mito.sequencing_group_id = sg.id
-    LEFT JOIN stripy on stripy.sequencing_group_id = sg.id
+    family f
+    LEFT JOIN family_participant fp ON f.id = fp.family_id
+    LEFT JOIN participant p ON fp.participant_id = p.id
+    LEFT JOIN sample s ON p.id = s.participant_id
+    LEFT JOIN sequencing_group sg on sg.sample_id = s.id
 WHERE
-    a.project IN :projects
+    f.project IN :projects
     AND sg.type IN :sequencing_types
-    AND a.status = 'COMPLETED'
-    AND a.type = 'WEB';
+ORDER BY
+    f.project,
+    sg.type,
+    sg.platform,
+    sg.technology,
+    s.type,
+    f.id,
+    fp.participant_id,
+    sg.id;
         """
 
-        sequencing_group_reports = await self.connection.fetch_all(
+        families = await self.connection.fetch_all(
             _query,
             {
-                'projects': projects,
-                'sequencing_types': sequencing_types,
+                "projects": project_ids,
+                "sequencing_types": sequencing_types,
             },
         )
-        return sequencing_group_reports
+        families_by_project_id_and_seq_fields = defaultdict(list)
+        for row in families:
+            families_by_project_id_and_seq_fields[
+                (
+                    row["project"],
+                    row["sequencing_type"],
+                    row["sequencing_platform"],
+                    row["sequencing_technology"],
+                )
+            ].append(row)
```

Comment:
Beware returning row pointer here, it's not a dict. In this case, it carries a lot of extra fields, and you need to know the keys by looking the code. Consider returning a FamilyInternal object, as you can leverage typing to understand exactly what the response looks like